operatorcourier/api.py

```diff
@@ -62,11 +68,11 @@ def build_verify_and_push(namespace, repository, revision, token, source_dir=Non
     :param yamls: List of yaml strings to create bundle with
     """
     
-    bundle = build_and_verify(source_dir, yamls)
+    bundle = build_and_verify(source_dir, yamls, validation_output=validation_output)
 
     if bundle is not None:
         with TemporaryDirectory() as temp_dir:
-            with open('%s/bundle.yaml' % temp_dir, 'w') as outfile:
+            with open('{}/bundle.yaml'.format(temp_dir), 'w') as outfile:
```

Comment:
Is this related to the change in the pr? why add this?

Reply:
in any case `os.path.join()` is better.